src/components/DestinationsColumns/columns.tsx

```diff
@@ -0,0 +1,177 @@
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '../ui/tooltip';
+
+import { AccessKey } from '@bsv/spv-wallet-js-client';
+import { Link } from '@tanstack/react-router';
+import { ColumnDef } from '@tanstack/react-table';
+
+import { ArrowUpDown } from 'lucide-react';
+
+import { Badge } from '@/components/ui/badge.tsx';
+
+import { Button } from '@/components/ui/button.tsx';
+
+export interface AccessKeysColumns extends AccessKey {
+  status: string;
+}
+
+export const columns: ColumnDef<AccessKeysColumns>[] = [
+  {
+    accessorKey: 'id',
+    header: ({ column }) => {
+      return (
+        <Link
+          from={'/destinations'}
+          search={(prev) => ({
+            ...prev,
+            order_by_field: 'id',
+            sort_direction: column.getIsSorted() === false ? 'asc' : (column.getIsSorted() as string),
```

Comment:
Once again 🙈 
Do we have a helper function for this? I guess we do 😄

Reply:
Of course we have  😄. These code was copied from previous iterations. Will fix.